project-base/translations/messages.cs.po

```diff
@@ -173,19 +173,19 @@ msgid "EAN"
 msgstr "EAN"
 
 msgid "Email"
-msgstr "E-mail"
+msgstr "Email"
 
 msgid "Email with a link to the export of your personal data was sent to your email address."
-msgstr "E-mail s odkazem na stránku exportu osobních údajů byl odeslán na vaši e-mailovou adresu."
+msgstr "Email s odkazem na stránku exportu osobních údajů byl odeslán na vaši emailovou adresu."
```

Comment:
Are you really sure about the Czech variant of word email? 
I think in Czech is used the variant with hyphen because email is kind of paint.
https://www.pravopisne.cz/2015/01/e-mail-x-email/

Reply:
fixed